src/services/BiometricService.ts

```diff
@@ -19,7 +15,7 @@ export default class BiometricService {
     this.setBiometricsAvailability();
   }
 
-  biometryType: BiometryType;
```

Comment:
it is correct ?

Reply:
I think so. As you can see, previously we were importing only `ReactNativeBiometrics` from `'react-native-biometrics'` which never has been correct - `ReactNativeBiometrics` simply doesn't have a types like `ReactNativeBiometrics.FaceID` and so on... So, I imported `BiometryType` which was actually something that we were looking for but because of the name conflict I had to rename it. 

Does it make sense?

Reply:
trust u 👍